schemas/event.yml

```diff
@@ -128,6 +128,27 @@
 
         In case the two timestamps are identical, @timestamp should be used.
 
+    - name: start
+      type: date
+      description: >
+        event.start contains the date when the event started.
+
+        This timestamp is distinct from @timestamp in that @timestamp contains
+        the processed timestamp. For logs these two timestamps can be different
+        as the timestamp in the log line and when the event is read for example
+        by Filebeat are not identical. `@timestamp` must contain the timestamp
+        extracted from the log line, event.start when the log line is read.
```

Comment:
For me that is `event.created`.

Reply:
I would not explain `event.start` in terms of the phenomenon of reading log files.

I think it would be more beneficial to specifically explain `start`/`end` specifically in terms of the observation of long running phenomena, like flows.

Some points I'd try to hit in this description and/or `end`:

* `start` and `end` usually go hand in hand. Although there may be exceptions (missing field) in the case of partial flows.
* These are to describe potentially long running events. `start` can actually be hours before `@timestamp`, if the observed phenomenon started a long time ago and was just reported.
* I would relate this description to `event.duration`. 

To @ruflin's point, having 2/3 fields out of `start`, `end` and `duration` is enough to derive the 3rd fields. I'd rather have 3/3, but we may (or may not) want to mention that 2/3 can be enough?